app/app.go

```diff
@@ -172,6 +189,10 @@ var (
 )
 
 func init() {
+	// Replace defaults
+	// manually update the power reduction by replacing micro (u) -> atto (a)
+	sdk.DefaultPowerReduction = cosmosevmtypes.MicroPowerReduction
```

Comment:
This should be the same, no?
Cause the[ DefaultPowerReduction on the sdk is 1000000](https://github.com/cosmos/cosmos-sdk/blob/e265bb9b42308dc70743b6200a70db9aafb70527/types/staking.go#L24)
and on the evm, the [MicroPowerReduction is also 1e6 (1000000)](https://github.com/cosmos/evm/blob/73bc2435c3886d7ae181b992a7e6dd2e1c04093b/types/power.go#L14)